src/main/python/tf/util/init.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import os
+import zipfile
+from pyspark import SparkConf, SparkContext, SQLContext
+
+
+def init_spark(master, aut_jar):
+    conf = SparkConf()
+    conf.set("spark.jars", aut_jar)
+    conf.set("spark.sql.execution.arrow.enabled", "true")
+    conf.set("spark.sql.execution.arrow.maxRecordsPerBatch", "1280")
+    conf.set("spark.executor.memory", "16G")
+    conf.set("spark.cores.max", "84")
+    conf.set("spark.executor.cores", "6")
+    conf.set("spark.driver.memory", "64G")
+    conf.set("spark.task.cpus", "3")
+    sc = SparkContext(master, "aut image analysis", conf=conf)
+    sql_context = SQLContext(sc)
+    return conf, sc, sql_context
+
+
+def get_args():
+    parser = argparse.ArgumentParser(description='PySpark for Web Archive Image Retrieval')
+    parser.add_argument('--web_archive', help='input directory for web archive data', default='/tuna1/scratch/nruest/geocites/warcs')
+    parser.add_argument('--aut_jar', help='aut compiled jar package', default='aut/target/aut-0.17.1-SNAPSHOT-fatjar.jar')
+    parser.add_argument('--aut_py', help='path to python package', default='aut/src/main/python')
```

Comment:
This is the path to AUT python package, in order to import `aut.common`. I found this is not necessary though. It's already hard coded in `detect.py` (line 5,6). So I think I just delete this to avoid confusion.